sdk/python/feast/infra/feature_servers/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM public.ecr.aws/lambda/python:3.9
+
+# Copy app handler code
+COPY sdk/python/feast/infra/feature_servers/aws_lambda/app.py ${LAMBDA_TASK_ROOT}
+
+# Copy necessary parts of the Feast codebase
+COPY sdk/python sdk/python
+COPY protos protos
+COPY README.md README.md
+
+# Install Feast for AWS with Lambda dependencies
+RUN pip3 install -e 'sdk/python[aws,redis]'
+RUN  pip3 install -r sdk/python/feast/infra/feature_servers/aws_lambda/requirements.txt --target "${LAMBDA_TASK_ROOT}"
+
+ENV FEAST_USAGE=False
```

Comment:
Why are we disabling usage?

Reply:
iirc it was slowing things down. We can enable it after the issue with usage logging is fixed.